scalafmt-core/shared/src/main/scala/org/scalafmt/internal/FormatOps.scala

```diff
@@ -823,8 +821,19 @@ class FormatOps(val tree: Tree, val initStyle: ScalafmtConfig) {
       if (isBracket) close // If we can fit the type params, make it so
       else lastParen // If we can fit all in one block, make it so
 
+    val maxArity = valueParamsOwner match {
+      case d: Decl.Def if d.paramss.nonEmpty => d.paramss.map(_.size).max
```

Comment:
`Init` came in scalameta v2 but scalafmt is still on v1,

Seems in v1 `Init` is nested `Term.Apply`

```scala
@ q"@a(1)(2) def foo = 1"
res1: Defn.Def = Defn.Def(List(Mod.Annot(Term.Apply(Term.Apply(Ctor.Ref.Name("a"), List(Lit.Int(1))), List(Lit.Int(2))))), Term.Name("foo"), List(), List(), None, Lit.Int(1))
```

It's fine to skip those as they only appear in curried annotations, which is quite rare.